src/Resources/config/services.yml

```diff
@@ -24,3 +24,12 @@ services:
     ezplatform.http_cache.tag_aware_store:
         class: EzSystems\PlatformHttpCacheBundle\Proxy\TagAwareStore
         arguments: ["%ezplatform.http_cache.store.root%"]
+
+    ezplatform.http_cache.tag_handler:
+        class: EzSystems\PlatformHttpCacheBundle\Handler\TagHandler
+        # use parent: ???
```

Comment:
Hmm, looks like a TODO :)
Answer: in this case probably not as you've overridden constructor to use our custom arguments / services

Reply:
Yeah... That line was never meant to be committed.. Fixed.